scripts/jenkins/groovy/defineTestStages.groovy

```diff
@@ -37,11 +37,6 @@ def call(final pipelineContext) {
       stageName: 'Flow Headless Smoke', target: 'test-flow-headless-smoke',timeoutValue: 20,
       component: pipelineContext.getBuildConfig().COMPONENT_JS
     ],
-    [
-      stageName: 'Java 7 Smoke', target: 'test-junit-7-smoke-jenkins', javaVersion: 7, timeoutValue: 20,
-      component: pipelineContext.getBuildConfig().COMPONENT_JAVA,
-      image: "${pipelineContext.getBuildConfig().DOCKER_REGISTRY}/opsh2oai/h2o-3/dev-openjdk-7:${pipelineContext.getBuildConfig().DEFAULT_IMAGE_VERSION_TAG}"
-    ],
```

Comment:
we can trust the compiler restrictions only for the language features but not for the changes in JDK. So, do you plan to have a Java 7 stage to test the MOJOs?

Reply:
We should do that, that is a good idea. I think MOJO Compatibility suite should definitely run with J7 as well.

Reply:
@seb-h2o Added a MOJO Compatibility test on Java 7.

Let's see if this PR passes :)